### PR TITLE
fix: Handle null acquired and no treasury in guild map [skip ci]

### PR DESCRIPTION
**Pull request by ShadowCat117**

### CODE REVIEW

**Comment by Essentuan** [comment]
Territories are always at least 'Very Low' (or at least should be) so I think it would be better to just replace none with vlow here. (unless this changed very recently none territories have gotten medium treasury before)

**Comment by ShadowCat117** [comment]
GuildResourceValues handles both treasury and defences, the problematic territory currently has a none treasury
image

**Comment by ShadowCat117** [comment]
API appears to be fixed now so it's no longer causing issues but still worth including this in case it happens again

**Comment by Essentuan** [comment]
weird

File: common/src/main/java/com/wynntils/models/territories/type/GuildResourceValues.java [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
- * Copyright © Wynntils 2022-2023.
+ * Copyright © Wynntils 2022-2024.
  * This file is released under LGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.models.territories.type;
 
 import net.minecraft.ChatFormatting;
 
 public enum GuildResourceValues {
+    NONE("None", ChatFormatting.WHITE, 0),
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/territories/type/GuildResourceValues.java:10]: Does the API actually return "NONE", or where does this get set?

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/territories/type/GuildResourceValues.java:10]: This is from the advancements menu I believe, which displays "None" as seen in the image I posted above

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/territories/type/GuildResourceValues.java:10]: Ah!

**Review by kristofbolyai — APPROVED** [review]
Wynn moment.

**Pull request merged**